rocks-db/src/asset_client.rs

```diff
@@ -113,6 +113,22 @@ impl Storage {
         );
         Ok(())
     }
+
+    pub fn clean_syncronized_idxs(
+        &self,
+        asset_type: AssetType,
+        last_synced_key: Vec<u8>,
+    ) -> Result<()> {
+        let cf = match asset_type {
+            AssetType::Fungible => self.fungible_assets_update_idx.handle(),
+            AssetType::NonFungible => self.assets_update_idx.handle(),
+        };
+
+        let from = vec![];
+        self.db.delete_range_cf(&cf, from, last_synced_key)?;
```

Comment:
Maybe add some additional metrics to track the execution time of this operation?